core/src/main/web/app/helpers/core.js

```diff
@@ -1033,6 +1053,22 @@
             callback(result);
           }
         });
+      },
+      colorize: function (theme) {
+        var colorizedElements = $("html");
+        var ca = colorizedElements.attr('class');
+        var classes = [];
+        if(ca && ca.length && ca.split){
+          ca = jQuery.trim(ca); /* strip leading and trailing spaces */
+          ca = ca.replace(/\s+/g,' '); /* remove double spaces */
```

Comment:
If the trimming and removing double spaces is required I think it needs a comment explaining why since it's pretty surprising.